src/test/java/org/opensearch/knn/index/FaissIT.java

```diff
@@ -267,6 +270,115 @@ public void testEndToEnd_whenMethodIsHNSWPQ_thenSucceed() {
         fail("Graphs are not getting evicted");
     }
 
+    @SneakyThrows
+    public void testEndToEnd_whenMethodIsHNSWSQFP16_thenSucceed() {
```

Comment:
I understand that you follow pattern from other method but `testEndToEnd` doesn't have much meaning. Would it be better to name is `testHNSWSQFP16_whenIndexedAndQueried_thenSucceed()`

Reply:
makes sense